sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -312,31 +301,13 @@ export class ServiceBusClient {
    * @param queueOrTopicName The name of a queue or topic to send messages to.
    */
   createSender(queueOrTopicName: string): Sender {
-    const clientEntityContext = ClientEntityContext.create(
-      queueOrTopicName,
+    return new SenderImpl(
       this._connectionContext,
-      `${queueOrTopicName}/${generate_uuid()}`
+      queueOrTopicName,
+      this._clientOptions.retryOptions
     );
-    return new SenderImpl(clientEntityContext, this._clientOptions.retryOptions);
   }
 
-  // /**
-  //  * Gets a SubscriptionRuleManager, which allows you to manage Service Bus subscription rules.
-  //  * More information about subscription rules can be found here: https://docs.microsoft.com/en-us/azure/service-bus-messaging/topic-filters
-  //  * @param topic The topic for the subscription.
-  //  * @param subscription The subscription.
-  //  */
-  // getSubscriptionRuleManager(topic: string, subscription: string): SubscriptionRuleManager {
```

Comment:
Is this becoming part of the ATOM client?

Reply:
Yes